shared/common-adapters/standard-screen.desktop.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React from 'react'
+import {Box, Text, Icon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props} from './standard-screen'
+
+const defaults = {
+  showClose: true,
+}
+
+const StandardScreen = (props: Props) => {
+  const {children, onClose, showClose, errorMessage} = {...defaults, ...props}
+  return (
+    <Box style={styleContainer}>
+      {!!onClose && showClose && <Icon style={styleClose} type='iconfont-close' onClick={() => onClose()} />}
+      {errorMessage && <Box style={styleErrorBanner}><Text style={styleErrorBannerText} type='BodySmallSemibold'>{errorMessage}</Text></Box>}
+      <Box style={styleContentContainer}>
+        {children}
+      </Box>
+    </Box>
+  )
+}
+
+const styleContainer = {
+  ...globalStyles.flexBoxColumn,
+  flex: 1,
+  alignItems: 'center',
+  position: 'relative',
+  paddingTop: globalMargins.large,
+  paddingBottom: globalMargins.large,
+  ...globalStyles.scrollable,
+}
+
+const styleClose = {
+  position: 'absolute',
+  top: globalMargins.small,
+  right: globalMargins.small,
+  ...globalStyles.clickable,
+  color: globalColors.black_10,
+}
+
+const styleErrorBanner = {
+  ...globalStyles.flexBoxColumn,
+  justifyContent: 'center',
+  alignItems: 'center',
+  width: '100%',
+  zIndex: 1,
+  height: globalMargins.large,
+  marginTop: -globalMargins.large,
+  backgroundColor: globalColors.red,
+}
+
+const styleErrorBannerText = {
+  color: globalColors.white,
+}
+
+const styleContentContainer = {
+  ...globalStyles.flexBoxColumn,
+  flex: 1,
+  justifyContent: 'center',
+  alignItems: 'center',
+  margin: globalMargins.large,
+  maxWidth: 512,
```

Comment:
How was this value chosen? I'm not sure if this component should define a `maxWidth` -- seems like the child's responsibility.